pkg/systemd/service-details.css

```diff
@@ -0,0 +1,63 @@
+.service-top-panel .service-name {
+    font-size: var(--pf-global--FontSize--3xl);
+    font-weight: lighter;
+}
+
+.service-top-panel .onoff-ct,
+.service-top-panel .fa-ellipsis-v {
+    vertical-align: text-bottom;
+}
+
+.service-top-panel .onoff-ct {
+    padding-left: 60px;
+}
+
+.action-button {
+    margin-left: 30px;
+}
+
+.form-table-ct td:first-child {
+    font-weight: bolder;
+}
+
+.new-section {
+    border-bottom: 20px solid transparent;
+}
+
+.padding-inherit {
+    padding: inherit;
+}
+
+.bigger-clickable-area {
+    padding: 5px 10px 5px 10px;
+}
+
+.status-icon {
+    padding-right: 5px;
+}
+
+.green-icon {
+    color: var(--pf-global--success-color--100);
```

Comment:
This color creates very little contrast against the white, so I wonder if `--pf-global--success-color--200` would be a better choice here.

Reply:
But that's more of a gut feeling on my side. Not sure how to check it against the [contrast ratios mentioned here](https://www.patternfly.org/v4/design-guidelines/styles/colors)

Reply:
Yeah, the colors and font sizes I picked were mostly random and I am open to change those :) I think this one is the only one that I picked because it is default green color for patternfly icons, but agree, I also think this is not green enough.

Reply:
Oh, interesting. Do you know where the guidelines for that is?
I was looking at this page, under Alert statuses > Success > Colors in Context

Reply:
I don't really know, I just picked it up when I imported `pficon-ok` from https://www.patternfly.org/styles/icons/ and with inspector looked at its color.

Reply:
I changed it to `--pf-global--success-color--200` but it does not seem that distinguishable which are green and which are grey.
![notso2](https://user-images.githubusercontent.com/12330670/60955878-ff98a780-a301-11e9-88f1-d50c414d6a10.png)

Reply:
But yeah, is less distractive, especially with red one:
![withred](https://user-images.githubusercontent.com/12330670/60956112-72098780-a302-11e9-8941-80eaeb9d0fc4.png)